src/core/components/NavBar.module.css

```diff
@@ -28,7 +28,7 @@
 .logo {
   display: flex;
   align-items: center;
-  padding: var(--spacing-md) var(--spacing-xl) var(--spacing-md) 0;
+  padding: var(--spacing-md) 0 var(--spacing-md) 0;
```

Comment:
The code here previously was a temporary adjustment to fill space until this NewChatIcon was added. Now it's adjusted to have correct spacing on the edges and space correctly with the other elements